src/vm/lib/tests_helper.c

```diff
@@ -0,0 +1,50 @@
+#include "tests_helper.h"
```

Comment:
This file is a test helper as the name implies and shouldn't be inside the VM source code.